docs/src/pages/docs/events.md

```diff
@@ -0,0 +1,200 @@
+# Events
+
+Events are an incredibly powerful feature in Anchor. Rather than polling or refreshing to
```

Comment:
Forgot to put this in the review, but this should be changed to support frontmatter like the rest of the articles I think. It should have a `title` and `description`.

Although I don't immediately see where the description is used, all of the other files have them.

Reply:
Done.